sdk/search/Azure.Search.Documents/CHANGELOG.md

```diff
@@ -77,6 +80,12 @@
 - Removed `SearchServiceCounters.SkillsetCounter`.
 - Removed `new SearchOptions(string continuationToken)` overload.
 
+## 11.0.0-preview.1 (2020-03-13)
```

Comment:
I think this is a regression in the CHANGELOG tooling because it used to respect previous entries.  Can you file a bug in https://github.com/Azure/azure-sdk-tools/ linking this and your release prep PR?  //fyi @chidozieononiwu and @weshaggard.

Reply:
I removed auto-merge because we don't want this going in.  Can you make a separate PR to bump the version correctly for Search?

Reply:
Pull request - https://github.com/Azure/azure-sdk-for-net/pull/18672

Reply:
Opened issue https://github.com/Azure/azure-sdk-tools/issues/1394

Reply:
I ran into this with KV yesterday and had opened https://github.com/Azure/azure-sdk-tools/issues/1389. I tend to reject these automatic PRs because of that but agree it should be fixed so it's not an issue.